test/unit/org/apache/cassandra/index/sai/IndexingSchemaLoader.java

```diff
@@ -121,7 +121,7 @@ public static TableMetadata.Builder clusteringSAITableMetadata(String ksName, St
         Indexes.Builder indexes = Indexes.builder();
         for (String indexedColumn : indexedColumns)
         {
-            indexes.add(IndexMetadata.fromSchemaMetadata(cfName + "_" + indexedColumn, IndexMetadata.Kind.CUSTOM, new HashMap<String, String>()
+            indexes.add(IndexMetadata.fromSchemaMetadata(cfName + '_' + indexedColumn, IndexMetadata.Kind.CUSTOM, new HashMap<String, String>()
```

Comment:
It's not related to the current changes, but I've realised that the only user of `IndexSchemaLoader` is `OperationTest`. So, unless we have short terms to add another users, we should merge both classes. 

Also, `IndexSchemaLoader` indexes multiple columns that are never mentioned on `OperationTest`. These are: `comment`, `comment_suffix_split`, `/output/full-name/`, `/data/output/id`, `first_name_prefix` and `nickname`. We probably should get rid of them for simplicity.

Finally, the block of code for adding and `IndexMetadata` to an `Index.Builder` is a bit reiterative. We could have an utility method such as:
```java
private static void adddIndex(Indexes.Builder indexes, String table, String column)
{
    String indexName = table + '_' + column;
    indexes.add(IndexMetadata.fromSchemaMetadata(indexName, IndexMetadata.Kind.CUSTOM, new HashMap<String, String>()
    {{
        put(IndexTarget.CUSTOM_INDEX_OPTION_NAME, StorageAttachedIndex.class.getName());
        put(IndexTarget.TARGET_OPTION_NAME, column);
    }}));
}
```